ReactAndroid/src/main/java/com/facebook/react/modules/camera/CameraRollManager.java

```diff
@@ -279,6 +286,19 @@ protected void doInBackgroundGuarded(Void... params) {
         selection.append(" AND " + SELECTION_BUCKET);
         selectionArgs.add(mGroupName);
       }
+      if (!TextUtils.isEmpty(mAssetType)) {
+        if(mAssetType.equals("Videos")) {
```

Comment:
Can you compare against a constant here instead of the string, "Videos"?

Reply:
Do i need to to upgrade my RN to 0.47.2. because below version doesn't have any video support

Reply:
This PR was not merged.

Reply:
I am using RN version 0.45.0. Video support is not there in 0.45.0. What should I do. Please let me know